server/utils/manifest.ts

```diff
@@ -0,0 +1,24 @@
+import fs from "fs";
+import path from "path";
+
+type Chunk = {
+  file: string;
+  src: string;
+  isEntry?: boolean;
+};
+
+type ManifestStructure = Record<string, Chunk>;
+
+const file = path.resolve("./build/app/manifest.json");
+
+let manifest = "{}";
+
+try {
+  manifest = fs.readFileSync(file, "utf8") as string;
+} catch (err) {
+  console.warn(
+    `Can not find ${file}. Try executing "yarn vite:build" before running in production mode.`
+  );
+}
+
+export default JSON.parse(manifest) as ManifestStructure;
```

Comment:
I didn’t saw there was code to read out the manifest.json already, but I like my implementation more - so I deleted the other one. Let me know if I should remove that and get the other one in again. 🙃